tests/operator/operator.go

```diff
@@ -2881,11 +2908,7 @@ spec:
 
 			var preferredTopology = v1beta1.PreferThreads
 
-			It("to instancetypes", func() {
-				By("Enabling the feature gate and waiting for KubeVirt to be ready")
```

Comment:
The FG should still be disabled by default, no?

Reply:
Yup sorry need to explicitly enable it [here](https://github.com/kubevirt/kubevirt/compare/d039d0493abf2fee428999fb36cbd7ead995eb6b..85ffefa2862f1918bc1a2e6be29cde590101dbd5).